release.yaml

```
@@ -0,0 +1,14 @@
+kuadrant-operator:
+  version: "0.0.0"
+
+olm:
+  channels: 
+  - "0.0.0"
+  default-channel: "0.0.0"
```

`default-channel` and `channels` first (and only so far) entry should be `alpha` I reckon, wdyt @eguzki @maleck13 

```suggestion
  channels: 
  - "alpha"
  default-channel: "alpha"
```

---

or `default-channel: "stable"` (?)